test/ILogger.NetStandard.Tests/ILoggerIntegrationTests.cs

```diff
@@ -0,0 +1,236 @@
+                (telemetryItem, telemetryProcessor) => itemsReceived.Add(telemetryItem),
+                configureTelemetryConfiguration: null,
+                configureApplicationInsightsOptions: (appInsightsLoggerOptions) => appInsightsLoggerOptions.TrackExceptionsAsExceptionTelemetry = false);
+
+            testLogger = serviceProvider.GetRequiredService<ILogger<ILoggerIntegrationTests>>();
+
+            testLogger.LogInformation("Testing");
+            testLogger.LogError(new Exception("TestException"), "Exception");
+
+            Assert.IsInstanceOfType(itemsReceived[0], typeof(TraceTelemetry));
+            Assert.IsInstanceOfType(itemsReceived[1], typeof(TraceTelemetry));
+
+            Assert.AreEqual(SeverityLevel.Information, (itemsReceived[0] as TraceTelemetry).SeverityLevel);
+            Assert.AreEqual("Testing", (itemsReceived[0] as TraceTelemetry).Message);
+
+            Assert.AreEqual(SeverityLevel.Error, (itemsReceived[1] as TraceTelemetry).SeverityLevel);
+            Assert.AreEqual("Exception", (itemsReceived[1] as TraceTelemetry).Message);
+        }
+
+        /// <summary>
+        /// Ensures that the <see cref="ApplicationInsightsLoggerOptions.IncludeScopes"/> switch is honored.
+        /// </summary>
+        [TestMethod]
+        [TestCategory("ILogger")]
+        public void ApplicationInsightsLoggerHonorsIncludeScopeSwitch()
+        {
+            List<ITelemetry> itemsReceived = new List<ITelemetry>();
+
+            // Case where Scope is included.
+            IServiceProvider serviceProvider = ILoggerIntegrationTests.SetupApplicationInsightsLoggerIntegration(
+                (telemetryItem, telemetryProcessor) => itemsReceived.Add(telemetryItem),
+                configureTelemetryConfiguration: null,
+                configureApplicationInsightsOptions: (appInsightsLoggerOptions) => appInsightsLoggerOptions.IncludeScopes = true);
+
+            ILogger<ILoggerIntegrationTests> testLogger = serviceProvider.GetRequiredService<ILogger<ILoggerIntegrationTests>>();
+
+            testLogger.BeginScope("TestScope");
+            testLogger.BeginScope<IReadOnlyCollection<KeyValuePair<string, object>>>(new Dictionary<string, object> { { "Key", "Value" } });
+
+            testLogger.LogInformation("Testing");
+            testLogger.LogError(new Exception("TestException"), "Exception");
+
+            Assert.AreEqual(" => TestScope", (itemsReceived[0] as ISupportProperties).Properties["Scope"]);
+            Assert.AreEqual("Value", (itemsReceived[0] as ISupportProperties).Properties["Key"]);
+
+            Assert.AreEqual(" => TestScope", (itemsReceived[1] as ISupportProperties).Properties["Scope"]);
+            Assert.AreEqual("Value", (itemsReceived[1] as ISupportProperties).Properties["Key"]);
+
+            itemsReceived.Clear();
+
+            // Case where Scope is NOT Included
+            serviceProvider = ILoggerIntegrationTests.SetupApplicationInsightsLoggerIntegration(
```

Comment:
Split this test and `ApplicationInsightsLoggerHonorsLogExceptionsAsExceptionTelemetrySwitch` into two.

Reply:
Done.